core/services/relay/evm/types/gen/ints.go.tmpl

```diff
@@ -1,3 +1,4 @@
+// this file is auto-generated. DO NOT EDIT
```

Comment:
This is a template file, not auto-generated right? Should we get rid of this message?

Reply:
Oh, is this in here so that it generates the message in the generated files?

We could write it to `outputFile` in main.go just before res gets written instead:
https://github.com/smartcontractkit/chainlink/blob/unaddressable-int256-type/core/services/relay/evm/types/gen/main.go#L54